stacks-signer/src/v0/signer.rs

```diff
@@ -90,7 +91,7 @@ pub struct Signer {
     /// marking a submitted block as invalid
     pub block_proposal_validation_timeout: Duration,
     /// The current submitted block proposal and its submission time
-    pub submitted_block_proposal: Option<(BlockProposal, Instant)>,
+    pub submitted_block_proposal: Option<(Sha512Trunc256Sum, Instant)>,
```

Comment:
@jferrant I've refactored this to just track the `signer_signature_hash`. I had started the SignerDB stuff to only track via signer_sig_hash, and so it was easier to integrate with this using just the sighash as well. The code using this property only ever used the sighash, also.